index.js

```diff
@@ -115,7 +122,11 @@ const init = ({
       shouldSkipTest = canSkipTest()
       if (shouldSkipTest) return
 
-      console[methodName] = newMethod // eslint-disable-line no-console
+      // eslint-disable-next-line no-console
+      console[methodName] = (...args) => {
+        newMethod(...args)
```

Comment:
Do you want to make this change the default behavior or only if the new option returns true?

I'm open to both but just want to ensure that the implementation matches the original intent.